utils/mock-data-loader/package-lock.json

```diff
@@ -1,437 +1,9 @@
 {
   "name": "mock-data-loader",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
this needs to be `2` . You may need to update your npm version to use the latest one